File: BasicInputOutput/C++/GreatestOfTwo.cpp
#include <iostream>
using namespace std;

int main() {
    float num1, num2;
    cout << "Enter Two Values: ";
    cin >> num1 >> num2;

    cout << ((num1 == num2) ? "Both Values are Equal" : (num1 > num2) ? "Num1 is greater" : "Num2 is greatest") << endl;

    return 0;
}

File: BasicInputOutput/Java/GreatestOfTwo.java

import java.util.Scanner;

class GreatestOfTwo{
    public static void main(String[] args) {
        Scanner sc  =new  Scanner(System.in);

        System.out.println("Enter Two Value ");
        float num1 = sc.nextFloat();
        float num2 = sc.nextFloat();

        System.out.println((num1==num2)?"Both VAlue is Equal" : (num1>num2)?"Num1 is greater" : "Num2 is greatest");
    }
}